edi_account_oca/tests/test_edi.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020 Creu Blanca
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import fields
+from odoo.tests.common import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.addons.component.core import Component
+from odoo.addons.component.tests.common import TransactionComponentRegistryCase
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged("-at_install", "post_install")
+class EDIBackendTestCase(AccountTestInvoicingCommon, TransactionComponentRegistryCase):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
```

Comment:
There has been a change here. You can read more about it at `https://github.com/odoo/odoo/pull/152899`
```suggestion
    def setUpClass(cls):
        super().setUpClass()
```